ploy_ezjail/__init__.py

```diff
@@ -174,6 +174,24 @@ def start(self, overrides=None):
             log.info("Instance state: %s", status)
             log.info("Instance already started")
             return True
+
+        rc_provide = self.config.get('rc_provide', '')
+        self.master._exec("sed",
+            "-i",
+            "",
+            "-e",
+            "s/\# PROVIDE: standard_ezjail/\# PROVIDE: standard_ezjail %s %s/" % (self.id, rc_provide),
+            "/usr/local/etc/ezjail/%s" % self.id)
+
+        rc_require = self.config.get('rc_require')
+        if rc_require is not None:
+            self.master._exec("sed",
+                "-i",
+                "",
+                "-e",
+                "s/\# REQUIRE:/\# REQUIRE: %s/" % rc_require,
```

Comment:
sed always replaces the whole line, so i can't see how this would add new lines or the list.

also, this is never set to anything since the file has just been created by ezjail a few lines above